rtsf-at-checkout-loss-detector/res/configuration.toml

```diff
@@ -14,7 +14,7 @@ Port = 48096
 Protocol = 'http'
 ReadMaxLimit = 100
 StartupMsg = 'Starting Loss Detector'
-Timeout = '30s'
+RequestTimeout = '30s'
```

Comment:
Rename `ClientMonitor ` to `HealthCheckInterval`
Remove `BootTimeout`
Remove `ClientMonitor` 
Remove `Protocol `
Missing these:
 - MaxRequestSize = 0
 - ServerBindAddr = ""
 - MaxResultCount = 0